src/createLambdaProxyContext.js

```diff
@@ -12,17 +12,18 @@ module.exports = function createLambdaProxyContext(request, options, stageVariab
 
   let body = request.payload;
 
-  const headers = request.unprocessedHeaders;
+  const headers = request.headers;
+  const unprocessedHeaders = request.unprocessedHeaders;
 
   if (body) {
     if (typeof body !== 'string') {
       body = JSON.stringify(body);
     }
-    headers['Content-Length'] = Buffer.byteLength(body);
+    unprocessedHeaders['content-length'] = Buffer.byteLength(body);
```

Comment:
When sending the following request
```
$ curl -X POST 'http://localhost:3000/hello' -d 'body'

"headers": {
			"Host": "localhost:3000",
			"User-Agent": "curl/7.59.0",
			"content-type": "application/x-www-form-urlencoded",
			"Accept": "*/*",
			"Content-Length": "6",
			"content-length": 4
		},
```

I debugged the request.headers object and this is how it's arriving
```
{ 
  host: 'localhost:3000',
  'user-agent': 'curl/7.59.0',
  accept: '*/*',
  'content-length': '4',
  'content-type': 'application/x-www-form-urlencoded' 
}
```
As unprocessed headers has the headers already, this line ads a new element in lower case.